t/geturl_ssl.pl

```diff
@@ -4,6 +4,7 @@
     require HTTP::Request;
     require HTTP::Response;
     require IO::Socket::IP;
+    require IO::Socket::SSL;
```

Comment:
Add `IO::Socket::SSL` to the optional test modules in `configure.ac`.

Reply:
Done